ccmlib/repository.py

```diff
@@ -540,8 +540,8 @@ def get_logger(log_file):
 
 
 def log_info(process, logger):
-    out = process.wait()
+    rc = process.returncode
     stdoutdata, stderrdata = process.communicate()
```

Comment:
You'll need to call communicate and _then_ check the returncode. Sorry for being wrong in my last comment